src/triton_gazebo/models/README.md

```diff
@@ -0,0 +1,35 @@
+
```

Comment:
I think this documentation should just be a section in the main triton_gazebo readme